### ISSUE TITLE
what should the type of `Message` look like?

### ISSUE THREAD

**Issue by samuelstroschein** [inlang/inlang#1111]
## Context

Only AST information is required for the pattern (body). Hence, simplify the type. 

## Proposal 

- id is only a string (no AST information needed)
- languageTag is included in message

### Proposal 1

- Rename pattern to body because wtf is a pattern? 

```ts

type Message = {
  id: string,
  languageTag: LanguageTag,
  body: Array<Text | Placeholder>
}

type Placeholder = VariableReference // | future placeholders

type Text = {
  type: "Text" 
  value: string
}

type VariableReference = {
  type: "VariableReference",
  name: string
}

```

### Proposal 2

- keep pattern name in alignment with upcoming message format 2.0 

```ts

type Message = {
  id: string,
  languageTag: LanguageTag,
  pattern: Array<Text | Placeholder>
}

// ...

```

**Comment by samuelstroschein** [comment]
Vote for proposal 1

**Comment by samuelstroschein** [comment]
ACtually, I tend towards proposal 2 because a pattern is indeed "unique". 

- a pattern consists of multiple elements that can be text or placeholders
- a pattern is not just text. 
- explaining pattern as a dedicated concept seems to make sense

**Comment by samuelstroschein** [comment]
@inlang/team

**Comment by samuelstroschein** [comment]
will be addressed in #1149
